text/0202-events-api.md

```diff
@@ -0,0 +1,227 @@
+# Introducing Events API
+
+
+In this OTEP we introduce an Events API that is based on the OpenTelemetry Log signal. In OpenTelemetry's perspective Log Records and Events are different names for the same concept - however, there is a subtle difference in how they are represented using the underlying data model that is described below. Although every language has APIs for logs, they are not all capable of creating events. We will describe why the existing Logging APIs are not sufficient for the purpose of creating events.  It will then be evident that we will need an API in OpenTelementry for creating events. 
```

Comment:
> In OpenTelemetry's perspective Log Records and Events are different names for the same concept

Where in the existing spec is this stated?  afaik Events are not a top-level entity, but a sub-entity within Spans.

Reply:
It is mentioned here - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/overview.md#events-and-logs